client/trino-cli/src/main/java/io/trino/cli/Console.java

```diff
@@ -71,14 +69,16 @@
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.jline.utils.AttributedStyle.CYAN;
 import static org.jline.utils.AttributedStyle.DEFAULT;
+import static picocli.CommandLine.PropertiesDefaultProvider;
 
 @Command(
         name = "trino",
         header = "Trino command line interface",
         synopsisHeading = "%nUSAGE:%n%n",
         optionListHeading = "%nOPTIONS:%n",
         usageHelpAutoWidth = true,
-        versionProvider = VersionProvider.class)
+        versionProvider = VersionProvider.class,
+        defaultValueProvider = PropertiesDefaultProvider.class)
```

Comment:
I somehow _totally_ missed there's already https://github.com/trinodb/trino/blob/master/client/trino-cli/src/main/java/io/trino/cli/Trino.java#L55

I'll remove this.